WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerViewModel.kt

```diff
@@ -43,10 +43,12 @@ import org.wordpress.android.ui.mediapicker.MediaPickerSetup.DataSource.STOCK_LI
 import org.wordpress.android.ui.mediapicker.MediaPickerSetup.DataSource.WP_LIBRARY
 import org.wordpress.android.ui.mediapicker.MediaPickerUiItem.ClickAction
 import org.wordpress.android.ui.mediapicker.MediaPickerUiItem.FileItem
+import org.wordpress.android.ui.mediapicker.MediaPickerUiItem.NextPageLoader
```

Comment:
Minor (🔍): Consider removing this import and reverting to the existing approach of using `MediaPickerUiItem.NextPageLoader` instead of just `NextPageLoader` as it saves us an import.

PS: Also, if you agree with the above please consider removing all `import org.wordpress.android.ui.mediapicker.MediaPickerUiItem.*` related imports and use `MediaPickerUiItem.Xyz` explicitly everywhere. This way we are saving us of having to declare 6 imports along with making the code more readable since it will not become more evident what this `ClickAction` refers to, and the rest of them, that is, that all are in reference to `MediaPickerUiItem`.

Reply:
👋🏾 @ParaskP7 made changes as per suggestion and pushed 🙌🏾

Reply:
Awesome, thank you for applying my suggestion! 🙇 

PS: Next time don't forget to mention the commit that fixes this comment as it will make it easier for the reviewer to resolve the converstation, like that: fb1d4869d17a037a9bc1d7e3b34fea9e17329b32